Ethereum/README.md

```diff
@@ -1,15 +1,174 @@
-# StraitsX-tokens
+# StraitsX-tokens (XSGD)
+
+## Token Description
+The StraitsX SGD (XSGD) token is the first digital token fully collateralized one-for-one with Singapore Dollar (SGD) and representing 1 SGD. The XSGD token is written as a smart contract on the Ethereum blockchain following the ERC20 standard and is governed by the StraitsX network whereby XSGD tokens are centrally minted and burned by Xfers Pte. Ltd. 
+
+The Xfers payment service is regarded as a Widely Accepted Stored Value Facility under Singapore law. Xfers Pte. Ltd., is the Approved Holder of the Xfers Wallet Stored Value Facility.
```

Comment:
This one can update to be for PSA already, we are a major payment institute

Reply:
What should I replace this with ah? 

`Xfers is a major payment institute under the Payments Services Act by the Monetary Authority of Singapore`?

Reply:
Will also need to update this on Zilliqa end

Reply:
Ok nvm got a nice one from the XSGD launch article 

`Xfers is a Major Payment Institution license (“MPI”) for e-money issuance licensed by the Monetary Authority of Singapore (MAS)`

Reply:
though i should replace "is a" with "is an approved holder of a"

Reply:
Oops yeah my bad for not comenting just now, anything you take from our launch article has been cleared and can be used readily